fab/services/templates/supervisor_celery_background_queue.conf

```diff
@@ -1,6 +1,6 @@
-[program:{{ project }}-{{ environment }}-celery_doc_deletion_queue]
+[program:{{ project }}-{{ environment }}-celery_background_queue]
 environment={{ supervisor_env_vars }}
-command={{ new_relic_command }}{{ virtualenv_root }}/bin/python {{ code_root }}/manage.py celery worker --queues=doc_deletion_queue --events --loglevel=INFO --hostname={{ host_string }}_doc_deletion_queue --maxtasksperchild=50 --concurrency={{ celery_params.doc_deletion_queue.concurrency }} -Ofair
+command={{ new_relic_command }}{{ virtualenv_root }}/bin/python {{ code_root }}/manage.py celery worker --queues=background_queue --events --loglevel=INFO --hostname={{ host_string }}_doc_deletion_queue --maxtasksperchild=5 --concurrency={{ celery_params.doc_deletion_queue.concurrency }} -Ofair
```

Comment:
doc_deletion_queue is still referenced in the hostname and concurrency params here